todoapp.socket

```diff
@@ -0,0 +1 @@
+`ListenStream=80`.
```

Comment:
This file is incomplete, best to just delete it unless you are going to complete Module 4 (we won't be returning to Ansible but we can give you a hand completing this stretch goal if you're interested)